pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -1713,6 +1717,16 @@ public void ShowProductTransitions()
             SetDisplayTypeChrom(DisplayTypeChrom.products);
         }
 
+        public void ShowRelativeAbundanceFormatting()
+        {
+            foreach (var summary in _listGraphPeakArea.Where(summary => summary.Type == GraphTypeSummary.abundance))
```

Comment:
Can there possibly ever be more than one of these? Maybe just find it using FirstOrDefault()